discovery/spec.md

```diff
@@ -924,52 +923,29 @@ Content-Type: application/json
 
 This MUST delete the Service at the referenced URL.
 
-The request MAY include a body.  Any such body MUST be an object containing an
-`id` matching the `{id}` in the path.  That object MAY include an `epoch`
-value.  If the Service corresponding with the specified `id` has a higher
-recorded `epoch` value than given, the request MUST fail.
-
-If other service attributes are included in the request, those MUST be
-ignored for the purposes of processing the request.
+The request URL MAY include an OPTIONAL `epoch` query parameter, and if
+present, the Discovery Endpoint MUST reject the request if the corresponding
+Service has a different recorded `epoch` value than the query parameter.
 
 The follow responses are defined by this specification:
-- `200 OK` if the new Service was deleted.
+- `200 OK` if the Service was deleted or does not exist.
   - The HTTP Response Body MUST include the Service values at the time of
     deletion.
-- `400 Bad Request` if `id` is in the body and does not match the `{id}` in the
-  path
-- `404 Not Found` if there is no Service with the specified `id`
-- `409 Conflict` if the given epoch was less than or equal to the existing
-  epoch of the Service with the given `id`
+- `409 Conflict` if the given epoch was not equal to the existing epoch of the
+  Service with the given `id`
 
 Other responses are allowed, but not defined by this specification.
 
-The format of the OPTIONAL HTTP Request body MUST adhere to the following:
-```
-Content-Type: application/json
```

Comment:
We have consistently supplied an example body, why remove it?

Reply:
See the comment above. In this variant of delete() where the ID is in the URL, the body serves no purpose other than to carry the epoch since the URL's ID and the body always the same.

Reply:
I understand your intention better now.  I still think that at times when coding that it's easier to use the body but I don't care much.  As you note, so long as the semantics of PUT and POST are mostly the same using POST would be fine to submit single records.  If the semantics of PUT will change, this is more relevant even if not terribly important since its just a form factor matter.